images/linux/scripts/installers/docker-moby.sh

```diff
@@ -16,6 +16,7 @@ if ! IsInstalled $docker_package; then
     apt-get remove -y moby-engine moby-cli
     apt-get update
     apt-get install -y moby-engine moby-cli
+    apt-get install --no-install-recommends -y moby-buildx
```

Comment:
1. Where do we have a simple test to validate moby-buildx plugin installation?
2. Where do we update the documentation file to include moby-buildx version?